src/mongo/db/encryption/encryption_options.h

```diff
@@ -54,6 +54,11 @@ struct EncryptionGlobalParams {
     std::string kmipClientCertificateFile;
     std::string kmipClientKeyFile;
     std::string kmipKeyIdentifier;
+    bool kmipRotateMasterKey{false};
+
+    constexpr bool shouldRotateMasterKey() const noexcept {
```

Comment:
What is the purpose of constexpr here? We don't have const EncryptionGlobalParams instances and we don't anticipate them. So this constexprt will be ignored.